scripts/provision-elasticsearch.sh

```diff
@@ -147,23 +159,29 @@ main () {
     # call appropriate provisioning functions:
     if [ "$os_distribution" = "Ubuntu" ]; then
         if [ "$os_release" = "14" ]; then
+            check_environment_variables
             provision_elasticsearch_ubuntu14
+            sudo install_plugins
```

Comment:
This does not work because `install_plugins` is a shell function. The `sudo` must be used inside it.  
You can use the same technique as e.g. `scripts/provision-web.sh`:

```
# runs as root or needs sudo?
if [[ "$EUID" -ne 0 ]]; then
    sudo='sudo'
else
    sudo=''
fi

[...]

install_plugins () {
     $sudo /usr/share/elasticsearch/bin/plugin install -b mapper-attachments
}
```